### ISSUE TITLE
Webpack as direct dependency

### ISSUE THREAD

**Issue by falafeljan** [pelagios/recogito2#648]
Right now, Webpack has to be installed globally via `npm i -g webpack webpack-cli`. This could be circumvented by installing it directly via as `devDependency` and creating an [npm script](https://docs.npmjs.com/misc/scripts). With that, we could reduce additional steps required during setup and pin webpack to particular versions for ensuring consistency.

That would entail adapting the sbt build scripts (for example, running `npm run dev` instead of `webpack --watch`).

I'm happy to submit a PR for that, but I'm unsure what effects that could have on existing installations. With Docker it's easy to implement, but what about Vagrant, for instance?

**Comment by rsimon** [comment]
I agree, and would be inclined to add that. Feel free to make the pull request.

One comment though: the Webpack stuff was only a sort-of intermediate phase, as far as the client-side code of Recogito is concerned.

- The majority of the code is old-school jQuery, using RequireJS for module loading (and the official Play/Require integration)
- All the new parts of the UI are now React projects in separate repositories:
  - [recogito2-workspace-frontend](https://github.com/pelagios/recogito2-workspace-frontend) for the document workspace and public profile page
  - [recogito2-admin-frontend](https://github.com/pelagios/recogito2-admin-frontend) for the administration interface
  - [recogito2-ui-components](https://github.com/pelagios/recogito2-ui-components) for various embedd components elsewhere (currently: the annotation stats page elements)
- The only component currently built via the webpack build in this repository is the [Bulk Annotation UI](https://github.com/pelagios/recogito2/tree/master/app/assets/javascripts/document/annotation/common/bulkannotation)

**Comment by falafeljan** [comment]
Alright, I see! I just noticed that the external dependency could be resolved quite easily with this. I'll create the PR soon 👍
